templates/tests/test-templates.sh

```diff
@@ -89,6 +89,9 @@ function deployTemplate {
     echo "Initializing template '$1' with branch '$2'"
     azd init -t "$1" -b "$2" -e "$3" --subscription "$4" --location "$5" --no-prompt
 
+    echo "Current value of USE_APIM: $USE_APIM"
+    azd env set USE_APIM true
```

Comment:
should it be
```suggestion
    azd env set USE_APIM $USE_APIM
```

??